tests/test_transformer.py

```diff
@@ -9,215 +9,291 @@
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
 import tempfile
-import shutil
-from os.path import abspath, join, dirname
+from os.path import abspath, dirname, join
 
 from preggy import expect
+from tornado.testing import gen_test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.testing'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)